#state x y angle
float32 state_x
float32 state_y
float32 state_angle
#motion vector dnf
int32 dnf_x
int32 dnf_y